site/app/routes/registry+/just.$name[.json].ts

```diff
@@ -0,0 +1,44 @@
+// http://localhost:3000/registry/just/breadcrumbs.json
+// https://sly-cli.fly.dev/registry/just/breadcrumbs.json
```

Comment:
Update the comment to use an item from this library – these are convenient helper comments you can cmd+click to open directly in the browser, which is super nice for debugging and development

```suggestion
// http://localhost:3000/registry/just/array-flatten.json
// https://sly-cli.fly.dev/registry/just/array-flatten.json
```